### PR TITLE
Add self-signed certificate steps to ShiftStack params module - OSASINFRA1122

### PR DESCRIPTION
**Pull request by maxwelldb**
Addresses https://issues.redhat.com/browse/OSDOCS-922

Awaiting:
- [x] Dev feedback
- [x] QE
- [x] Peer review

### CODE REVIEW

**Comment by iamemilio** [comment]
lgtm

**Comment by maxwelldb** [comment]
@iamemilio Thanks!

File: modules/installation-osp-describing-cloud-parameters.adoc [hunk 1]
```diff
@@ -40,8 +41,27 @@ clouds:
...
+. _Optional_: If your {rh-openstack} installation uses self-signed certificate authority (CA) certificates for endpoint authentication:
+.. Copy the certificate authority file to your machine.
+.. In the command line, run the following commands to add the machine to the certificate authority trust bundle:
++
+----
+$ sudo cp ca.crt.pem /etc/pki/ca-trust/source/anchors/
+$ sudo update-ca-trust extract
+----
+
+.. Add the `cacerts` key to `clouds.yaml`. The value must be a non-root accessible path to the CA certificate:
```
Comment (pierreprinetti) [inline modules/installation-osp-describing-cloud-parameters.adoc]: I am confused by the path being relative. What does it have to be relative to?

```
a non-root accessible path to the CA certificate, absolute or relative to [...]
```

@iamemilio relative to what? To the clouds.yaml file? Or to CWD while running the installer?

Comment (maxwelldb) [reply modules/installation-osp-describing-cloud-parameters.adoc]: My assumption was the working directory, but good catch! It's not clear.

Comment (iamemilio) [reply modules/installation-osp-describing-cloud-parameters.adoc]: It should be able to parse relative paths from the installer directory, but I think its safest to just use absolute paths

Comment (pierreprinetti) [reply modules/installation-osp-describing-cloud-parameters.adoc]: I have run a couple tests. These are my findings: https://github.com/openshift/installer/pull/3356

**Comment by mandre** [comment]
Beside the relative path issue that @pierreprinetti spotted, this new doc looks good. Thanks @maxwelldb.

**Comment by mandre** [comment]
Looks great! :rocket:

**Comment by maxwelldb** [comment]
Thanks! Squashed down to 1 commit. Awaiting QE.

**Comment by maxwelldb** [comment]
QE moved Jira ticket to Verified.

**Comment by maxwelldb** [comment]
Holding off on peer review based on https://issues.redhat.com/browse/OSASINFRA-1122?focusedCommentId=14015179&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14015179

**Comment by maxwelldb** [comment]
@morenod Any objections to removing the note here? https://github.com/openshift/openshift-docs/pull/20355/commits/7e867db8450e23d2c599713f012de828e0ddee49

**Comment by morenod** [comment]
lgtm

**Comment by maxwelldb** [comment]
@morenod Thanks!

**Comment by maxwelldb** [comment]
@morenod One last addition: are you able to +1 the note in 8d4e0a6?

**Comment by morenod** [comment]
> @morenod One last addition: are you able to +1 the note in [8d4e0a6](https://github.com/openshift/openshift-docs/commit/8d4e0a6d5e76cbbc13f117958241ee2ca1b4e4e2)?

Perfect, rotation now works fine

**Review by morenod — APPROVED** [review]

**Comment by maxwelldb** [comment]
Thanks!

File: modules/installation-osp-describing-cloud-parameters.adoc [hunk 2]
```diff
@@ -41,7 +42,34 @@ clouds:
       auth_url: 'https://10.10.14.22:5001/v2.0'
 ----
 
-. Place the file that you generate in one of the following locations:
+. _Optional_: If your {rh-openstack} installation uses self-signed certificate authority (CA) certificates for endpoint authentication:
```
Comment (kalexand-rh) [inline modules/installation-osp-describing-cloud-parameters.adoc:45]: We don't add styling to "Optional."

If you have to do this process if you meet these criteria, it's not optional.

File: modules/installation-osp-describing-cloud-parameters.adoc [hunk 3]
```diff
@@ -41,7 +42,34 @@ clouds:
...
+.. Copy the certificate authority file to your machine.
+.. In the command line, run the following commands to add the machine to the certificate authority trust bundle:
++
+----
+$ sudo cp ca.crt.pem /etc/pki/ca-trust/source/anchors/
+$ sudo update-ca-trust extract
+----
+
+.. Add the `cacerts` key to `clouds.yaml`. The value must be an absolute, non-root-accessible path to the CA certificate:
```
Comment (kalexand-rh) [inline modules/installation-osp-describing-cloud-parameters.adoc:54]: the `clouds.yaml` file.

File: modules/installation-osp-describing-cloud-parameters.adoc [hunk 4]
```diff
@@ -41,7 +42,34 @@ clouds:
...
++
+[TIP]
+====
+After you run the installer with a custom CA certificate, you can update the certificate by editing the value of the `ca-cert.pem` key in the `cloud-provider-config` keymap. On a command line, run:
+----
+$ oc edit configmap -n openshift-config cloud-provider-config
+----
+====
+
+. Place `clouds.yaml` in one of the following locations:
```
Comment (kalexand-rh) [inline modules/installation-osp-describing-cloud-parameters.adoc:72]: see above

**Review by kalexand-rh — COMMENTED** [review]
Couple of picks, then it LGTM.

**Comment by maxwelldb** [comment]
@kalexand-rh Done. Thanks!

**Pull request merged**